jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/util/GenericErrorSummaryCountCell.java

```diff
@@ -64,37 +71,41 @@ public void render(Context context,
         } else {
             tooltipClasses.remove(ERROR_PRESENT_STYLE);
         }
-
-        if (viewPresenter.getViewErrorsActionCondition().test(value)) {
+        
+        if (viewPresenter.isUserAuthorizedForPerspective(EXECUTION_ERRORS) && errCount > 0) {
             tooltip.setTitle(Constants.INSTANCE.ErrorCountNumberView(errCount));
-            tooltip.setAttribute(PROCESS_INSTANCE_ATTRIBUTE,
-                                 Long.toString(value.getProcessInstanceId()));
+            tooltip.setAttribute(DATA_INSTANCE_ATTRIBUTE,
+                                 value.getId().toString());
             tooltipClasses.add(LINK_AVAILABLE_STYLE);
 
             initTooltipsAsync();
 
         } else {
             tooltip.removeAttribute("title");
-            tooltip.removeAttribute(PROCESS_INSTANCE_ATTRIBUTE);
+            tooltip.removeAttribute(DATA_INSTANCE_ATTRIBUTE);
             tooltipClasses.remove(LINK_AVAILABLE_STYLE);
         }
 
         sb.appendHtmlConstant(tooltip.getOuterHTML());
     }
 
-    public void openErrorView(final String pid) {
-        viewPresenter.openErrorView(pid);
+    public void openErrorView(final String id){
+        final PlaceRequest request = new DefaultPlaceRequest(EXECUTION_ERRORS);
+        
+        request.addParameter(SEARCH_PARAMETER_IS_ERROR_ACK,
+                             Boolean.toString(false));
+        viewPresenter.openView(request,id);
```

Comment:
@mcivantos-tribalyte I still think that all of these should be moved to the presenter, in here you should only call the _openErrorView_ in the presenter.

Reply:
+1